### ISSUE TITLE
status_code: 500, correlation ID: *******-*********

### ISSUE THREAD

**Issue by KannamSridharKumar** [run-llama/llama_extract#30]
Running the example code provided in the documentation or notebooks using pedantic or json schema is giving the error: status_code: 500, correlation ID: *******-*********

Only the default schema created by the llama_extract is working. 

For example this code isn't working

from pydantic import BaseModel, Field

extractor = LlamaExtract()

class ResumeMetadata(BaseModel):
    """Resume metadata."""

    years_of_experience: int 
    highest_degree: str 
    professional_summary: str 
    
extraction_schema = await  extractor.acreate_schema("Test Schema", ResumeMetadata)

**Comment by n400peanuts** [comment]
Hi, I have the same issue, see [#21](https://github.com/run-llama/llama_extract/issues/21)

**Comment by DennisKuijs** [comment]
Yup, I also experience the same error when using the cURL commands to interact with the REST API

**Comment by neerajprad** [comment]
The previous was an experiment which we took down due to various stability issues. We have a new version of LlamaExtract out which we'll be happy to have you try out if you are interested.
